tst/EnergyPlus/unit/HVACFourPipeBeam.unit.cc

```diff
@@ -0,0 +1,1705 @@
+		"    CoolSys1 Demand Bypass Branch,  !- Name",
+		"    ,                        !- Maximum Flow Rate {m3/s}",
+		"    ,                        !- Pressure Drop Curve Name",
+		"    Pipe:Adiabatic,          !- Component 1 Object Type",
+		"    CoolSys1 Demand Bypass Pipe,  !- Component 1 Name",
+		"    CoolSys1 Demand Bypass Pipe Inlet Node,  !- Component 1 Inlet Node Name",
+		"    CoolSys1 Demand Bypass Pipe Outlet Node,  !- Component 1 Outlet Node Name",
+		"    Bypass;                  !- Component 1 Branch Control Type",
+		
+		"  Sizing:Plant,",
+		"    HeatSys1 Loop,                !- Plant or Condenser Loop Name",
+		"    Heating,                 !- Loop Type",
+		"    60.0,                    !- Design Loop Exit Temperature {C}",
+		"    11.1,                    !- Loop Design Temperature Difference {deltaC}",
+		"    Coincident,              !- Sizing Option",
+		"    2,                       !- Zone Timesteps in Averaging Window",
+		"    None;                    !- Coincident Sizing Factor Mode    ",
+
+		"  PlantLoop,",
+		"    HeatSys1 Loop,             !- Name",
+		"    WATER,                   !- Fluid Type",
+		"    ,                        !- User Defined Fluid Type",
+		"    HeatSys1 Loop Operation,   !- Plant Equipment Operation Scheme Name",
+		"    HeatSys1 Supply Outlet Node,  !- Loop Temperature Setpoint Node Name",
+		"    100,                     !- Maximum Loop Temperature {C}",
+		"    3,                       !- Minimum Loop Temperature {C}",
+		"    0.003,                   !- Maximum Loop Flow Rate {m3/s}",
+		"    0,                       !- Minimum Loop Flow Rate {m3/s}",
+		"    autocalculate,           !- Plant Loop Volume {m3}",
+		"    HeatSys1 Supply Inlet Node,!- Plant Side Inlet Node Name",
+		"    HeatSys1 Supply Outlet Node,  !- Plant Side Outlet Node Name",
+		"    HeatSys1 Supply Branches,  !- Plant Side Branch List Name",
+		"    HeatSys1 Supply Connectors,!- Plant Side Connector List Name",
+		"    HeatSys1 Demand Inlet Node,!- Demand Side Inlet Node Name",
+		"    HeatSys1 Demand Outlet Node,  !- Demand Side Outlet Node Name",
+		"    HeatSys1 Demand Branches,  !- Demand Side Branch List Name",
+		"    HeatSys1 Demand Connectors,!- Demand Side Connector List Name",
+		"    OPTIMAL;                 !- Load Distribution Scheme",
+
+		"  SetpointManager:Scheduled,",
+		"    HeatSys1 Loop Setpoint Manager,  !- Name",
+		"    Temperature,             !- Control Variable",
+		"    HeatSys1 Loop Temp Sch,    !- Schedule Name",
+		"    HeatSys1 Loop Setpoint Node List;  !- Setpoint Node or NodeList Name",
+
+		"  Schedule:Compact,",
+		"    HeatSys1 Loop Temp Sch,   !- Name",
+		"    Temperature,             !- Schedule Type Limits Name",
+		"    Through: 12/31,          !- Field 1",
+		"    For: AllDays,            !- Field 2",
+		"    Until: 24:00,60.0;       !- Field 3",
+
+		"  NodeList,",
+		"    HeatSys1 Loop Setpoint Node List,  !- Name",
+		"    HeatSys1 Supply Outlet Node;  !- Node 1 Name",
```

Comment:
@EnergyArchmage Was this required for your unit test to work properly? This could be something that I could possibly add into the process_idf() function.